numpy/lib/tests/test_io.py

```diff
@@ -574,16 +575,26 @@ def test_unicode_and_bytes_fmt(self, fmt, iotype):
     @pytest.mark.slow
     @requires_memory(free_bytes=7e9)
     def test_large_zip(self):
-        def check_large_zip():
+        def check_large_zip(memoryerror_raised):
+            memoryerror_raised.value=False
```

Comment:
nit: 
```suggestion
            memoryerror_raised.value = False
```

Reply:
Oops, of course! I don't know how they slipped by. Fixed.